packages/cli/src/commands/hydrogen/dev.ts

```diff
@@ -249,6 +249,7 @@ async function runDev({
             type: 0,
             message:
               'MiniOxygen cannot start because the server bundle has not been generated.',
+            skipOclifErrorHandling: true,
```

Comment:
Why is this needed here and in other commands? 🤔

Reply:
This seems to be a required parameter in cli-kit 3.51.0 for `FatalError` https://github.com/Shopify/cli/blob/9d4c411176e13edcf352e4a4d5341aedaf1ee476/packages/cli-kit/src/public/node/error.ts#L27

Reply:
Oh I see! I've seen there's a new `renderError` function. Maybe we should be using that instead of `renderFatalError`. Not a blocker for this PR though

Reply:
I think `renderFatalError` is still more appropriate in this command, as it accepts an `error` and renders it appropriately, whereas `renderError` needs a `headLine` and `body` etc.